### PR TITLE
Conscious Language Changes

### PR DESCRIPTION
**Pull request by srijan-sivakumar**
PR addressing the changes to be made in case
of presence of problematic language in the
documentation.

Fixes: #619
Signed-off-by: srijan-sivakumar <ssivakum@redhat.com>

### CODE REVIEW

File: docs/Administrator-Guide/Events-APIs.md [hunk 1]
```diff
@@ -320,32 +320,32 @@ BRICK_REPLACE                  | volume               | Volume Name
...
+                               | secondary            | Secondary Details(Secondaryhost::SecondaryVolume)
                                | no_verify            | No verify option is used or not
                                | push_pem             | Push pem option is used or Not
                                | ssh_port             | If SSH port is configured during Session Create
-                               | master               | Master Volume Name
+                               | primary              | Primary Volume Name
 GEOREP_START                   | force                | Force option used during session Start
-                               | master               | Master Volume Name
-                               | slave                | Slave Details(Slavehost::SlaveVolume)
+                               | Primary              | Primary Volume Name
```
Comment (atinmu) [inline docs/Administrator-Guide/Events-APIs.md:329]: Why is it in caps?

File: docs/Developer-guide/Backport-Guidelines.md [hunk 2]
```diff
@@ -1,18 +1,18 @@
 # Backport Guidelines
 In GlusterFS project, as a policy, any new change, bug fix, etc., are to be
-fixed in 'master' branch before release branches. When a bug is fixed in
-the master branch, it might be desirable or necessary in release branch.
+fixed in 'devel' branch before release branches. When a bug is fixed in
```
Comment (atinmu) [inline docs/Developer-guide/Backport-Guidelines.md:3]: are you going to name this branch to devel or main? github recommends it to be main though...

Comment (srijan-sivakumar) [reply docs/Developer-guide/Backport-Guidelines.md:3]: At present, the default branch of glusterfs repo is `devel`, I think that was done to indicate that it is a continuous development branch. But yes, github does recommend `main` and even the default branch of glusterdocs is named `main`. @sankarshanmukhopadhyay @amarts  any thoughts on this ?

**Comment by srijan-sivakumar** [comment]
@itisravi @sankarshanmukhopadhyay @Shwetha-Acharya could you review the changes. TIA.

File: docs/Developer-guide/Backport-Guidelines.md (hunk 2 above)
Comment (sankarshanmukhopadhyay) [reply docs/Developer-guide/Backport-Guidelines.md:3]: If this is the same topic we had a brief interaction on the Slack channel then I'd opt for main and devel. The former being a replacement for the term 'master'

**Comment by sankarshanmukhopadhyay** [comment]
@atinmu @amarts - my suggestion would be to use 'main' instead of 'master' branch and keep the 'devel' nomenclature as is. If this is found agreeable then this PR needs minor changes and it should be good to go.

@aravindavk - can you confirm that the name changes to the variables etc are present in the actual code as well ie. the documentation and code/configuration files/options are in sync?

File: docs/Developer-guide/Backport-Guidelines.md (hunk 2 above)
Comment (amarts) [reply docs/Developer-guide/Backport-Guidelines.md:3]: For glusterfs repo, changing branch name again would be an issue. So, this document is fine. For all other projects which are not gone through branch changes, I am fine with `main` as in future the default branch would be `main` in all repos.

**Review by itisravi — COMMENTED** [review]
+1

**Review by aravindavk — APPROVED** [review]
LGTM

**Review by kshithijiyer — APPROVED** [review]

**Pull request merged**